src/components/Settings/DatasourceSettings/index.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import ListItem from 'react-md/lib/Lists/ListItem';
```

Comment:
It's usually more comfortable to read imports cluster when they are divided according to commonality like:

import generic/external dependencies
[space]
import react-md dependencies (since there are usually many of them)
[space]
impoer project internal dependencies
[space]
export class ...